configtx/application_test.go

```diff
@@ -361,7 +361,7 @@ func TestAddAnchorPeer(t *testing.T) {
 	err = c.AddAnchorPeer("Org2", newOrg2AnchorPeer)
 	gt.Expect(err).NotTo(HaveOccurred())
 
-	gt.Expect(proto.Equal(c.UpdatedConfig(), expectedUpdatedConfig)).To(BeTrue())
+	gt.Expect(proto.Equal(c.UpdatedConfig().Config, expectedUpdatedConfig)).To(BeTrue())
```

Comment:
As a follow-on, I'd suggest writing a matcher and using it.

https://github.com/onsi/gomega/issues/292#issue-342933845